foodsaving/groups/stats.py

```diff
@@ -95,17 +95,17 @@ def get_group_members_stats(group):
     }]
 
 
-def get_group_stores_stats(group):
+def get_group_places_stats(group):
     fields = {
-        'count_total': group.store.count(),
+        'count_total': group.place.count(),
     }
 
-    for entry in group.store.values('status').annotate(count=Count('status')):
-        # record one value per store status too
+    for entry in group.place.values('status').annotate(count=Count('status')):
+        # record one value per place status too
         fields['count_status_{}'.format(entry['status'])] = entry['count']
 
     return [{
-        'measurement': 'karrot.group.stores',
+        'measurement': 'karrot.group.places',
```

Comment:
We should run something in influxdb to migrate measurements, otherwise stats become even more unreliable ;)

Reply:
Ah yes, influxdb db too. Also the tag names.

Reply:
It would need exporting, modifying, and reimporting, the export is like this:

```
influx_inspect export -database karrot-dev -datadir /var/lib/influxdb/data -waldir /var/lib/influxdb/wal -out /tmp/karrot-dev-export
```

A bit of find/replace, then a reimport (see https://docs.influxdata.com/influxdb/v1.7/tools/shell/#import-data-from-a-file-with-import). Didn't try that one.

Doesn't seem too hard, and would prefer that to having incorrect "legacy" names in the stats.

Reply:
I tried it out, exporting fstool on yuca and importing locally, seemed ok and didn't take very long either.

```
influx_inspect export -database fstool -datadir /var/lib/influxdb/data -waldir /var/lib/influxdb/wal -out /tmp/fstool_export -compress
```
... copy file to my laptop ...

```
zcat fstool_export | sed 's/store/place/g' > fstool_export_place
influx -import -path=fstool_export_place
```

Import only took about a minute. Could also rename the db to "karrot-world" whilst we're there...

Incidentally notice the warning:
```
[I] 2018-11-07T18:58:45Z WARN: 100% of max-values-per-tag limit exceeded: (100000/100000), db=fstool shard=164 measurement=django_request tag=full_path service=store
```

Reply:
See here for more details about the warning: https://github.com/bitlabstudio/django-influxdb-metrics/issues/27
The problem won't occur anymore in my fork of django-influxdb-metrics that we use since a few weeks.